### DISCUSSION TITLE
Nomination of @J-N-K as a distro maintainer

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-distro#1502] [Distro Maintainers Discussions]
Dear @openhab/distro-maintainers,

I would like to nominate @J-N-K for joining the distro maintainers team. He is the driving force behind openHAB core nowadays and has proven his expertise in manifold ways. Since the distro repo is often closely related to changes in openHAB core, I think it would be a natural step to welcome him here as well.

Please cast your vote by adding 👍 or 👎  to this nomination.

**Comment by kaikreuzer** [comment]
Wow, that was a very quick vote - thank you!
Welcome @J-N-K!

**Reply by J-N-K** [reply]
Thanks!
